images/helm-operator/tests/main.tf

```
@@ -0,0 +1,59 @@
+terraform {
+  required_providers {
+    oci       = { source = "chainguard-dev/oci" }
+    helm      = { source = "hashicorp/helm" }
+    imagetest = { source = "chainguard-dev/imagetest" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" {
+  input = var.digest
+}
+
+data "imagetest_inventory" "this" {}
+
+resource "imagetest_harness_k3s" "this" {
+  name      = "helm-operator"
+  inventory = data.imagetest_inventory.this
+
+  sandbox = {
+    mounts = [
+      {
+        source      = path.module
+        destination = "/tests"
+      }
+    ]
+  }
+}
+
+module "helm" {
+  source = "../../../tflib/imagetest/helm"
+  chart  = "https://bsgrigorov.github.io/helm-operator/helm-operator-0.0.2.tgz"
+  name   = "helm-operator"
+  values = {
+    image = {
+      operator = "data.oci_string.ref.registry_repo:data.oci_string.ref.pseudo_tag"
```

I think this isn't doing what you expected. You'll need to use the Terraform string format syntax here:
```suggestion
      operator = "${data.oci_string.ref.registry_repo}:${data.oci_string.ref.pseudo_tag}"
```